target/linux/ath79/dts/qca9558_linksys_ea4500-v3.dts

```diff
@@ -136,7 +136,7 @@
 
 			partition@400000 {
 				label = "ubi";
-				reg = <0x400000 0x4c00000>;
+				reg = <0x400000 0x7940000>;
```

Comment:
Oh, I forgot it. Please update makefile `IMAGE_SIZE`.

Reply:
Done.